packages/runtime/src/lesson-files.ts

```diff
@@ -63,7 +65,7 @@ export class LessonFilesFetcher {
     this._templateLoadTask?.cancel();
 
     const task = newTask(async (signal) => {
-      const response = await fetch(`/${templatePathname}`, { signal });
+      const response = await fetch(`${this._baseURL}${templatePathname}`, { signal });
```

Comment:
Yep that's the assumption. I'll add a comment to the constructor about the assumption. Also I think I was wrong to call this `baseURL` it's more `basePathname` as it's not a URL.

I don't think we can use `new URL` here because of that.

Reply:
I don't like making assumptions tbqh, especially not with things like this. As I said, can't we use `new URL()` or `path.join()` or something?

Reply:
No we can't because it's not an URL. We can't use `path` because this code is not in node. And the `joinPaths` utility is not available in this package.